app/views/vacancies/search/_sort.html.slim

```diff
@@ -0,0 +1,4 @@
+.govuk-grid-row
+  .govuk-grid-column-full
```

Comment:
Didn't we say that we should not use `govuk-grid-column-full` when you have something that takes the full row?